rke/k8s_version_info.go

```diff
@@ -574,6 +578,10 @@ func loadK8sVersionInfo() map[string]v3.K8sVersionInfo {
 			MinRancherVersion: "2.6.3-rc0",
 			MinRKEVersion:     "1.3.3-rc0",
 		},
+		"v1.22.5-rancher2-1": {
```

Comment:
Just wondering, do you know why we don't have any of v1.21 versions in this list?

Reply:
I had a discussion with Kinara, according to her 1.21 was introduced starting 2.6.0, so it needed to be present in all rancher versions (there was no min/max restrictions) so there's no min info in dev-v2.6 for 1.21. But we'll add it in Jan patches KDM PR.